docs/REST.md

```diff
@@ -64,14 +64,31 @@ Create a model.
 ### Parameters
 
 * `name`: The name of the model
+* `description`: The description of the model
+* `manufacturer`: The manufacturer of the model
+* `vendor`: The vendor who sold the model
+* `location`: Location where the model is stored
+* `isFaulty`: Whether the model is faulty of not
+* `faultDescription`: Description of the fault
+* `price`: Price of one model
+* `count`: Amount of this model in stock
 
 ### Sample Response
 
 ```json
 {
     "status": "success",
     "data": {
-        "address": "m8y7nEtAe"
+        "address": "m8y7nEtAe",
+        "name": "Resistor",
+        "description": "V = IR",
+        "manufacturer": "Pancakes R\' Us",
+        "vendor": "Mouzer",
+        "location": "Shelf 14",
+        "isFaulty": "false",
+        "faultDescription": "",
+        "price": "10.50",
```

Comment:
Should this be a string `"10.50"` or just the number `10.5`? JSON supports numbers.